SendLog.py

```diff
@@ -93,15 +111,74 @@ def get_line_and_state(file):
 		# Add figure ID to list
 		figure_list.append(figure_id)
 		return (None, INITIALIZATION)
+        elif state == BLOCK_IDENTIFICATION:#check for block identification 
+                return (line,BLOCK_IDENTIFICATION) 
 	elif state == ENDING:
 		# End of figure
 		# Remove figure ID from list
 		figure_list.remove(figure_id)
 		return (None, ENDING)
 	return (line, DATA)
+     
+def get_line_and_state(file, count):
+
+    #  Return the line from the log filebased on the line count 
+
+    # Function to get a new line from file
+    # This also parses the line and appends new figures to figure List
+    global figure_list
+    line = file.readlines()
+
+    if not line[count]: # If required line is not present
+        return (None, NOLINE)
+
+    parse_result = parse_line(line[count])
+    figure_id = parse_result[0]
+    state = parse_result[1]
+    if state == INITIALIZATION:
+        # New figure created
+        # Add figure ID to list
+        figure_list.append(figure_id)
+        return (None, INITIALIZATION)
+    elif state == BLOCK_IDENTIFICATION:#check for block identification 
+            return (str(figure_id),BLOCK_IDENTIFICATION)
```

Comment:
Inconsistent indentation as compared to the lines above and below. Please fix and avoid such inconsistencies